srcCppCommon/ParameterManager.cxx

```diff
@@ -394,6 +394,48 @@ void ParameterManager::initialize()
     create("useLegacyDynamicData", useLegacyDynamicData);
   #endif
 
+    Parameter<int> *sendQueueSize = new Parameter<int>(50);
+    sendQueueSize->set_command_line_argument("-sendQueueSize", "<number>");
+    sendQueueSize->set_description(
+            "Sets number of samples (or batches) in send\nqueue. Default: 50");
```

Comment:
As in the rest of the descriptions, leave the '\n' as the last character of the line like:
```
"Sets number of samples (or batches) in send\n"
"queue. Default: 50"
```

Reply:
That makes sense, thanks